src/api/routes.py

```
@@ -0,0 +1,141 @@
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Server '{server_name}' not found",
+        )
+    
+    del mcp_config[server_name]
+    config_manager.update_config_value("mcp", "mcpServers", mcp_config)
+    
+    return {"message": f"Server '{server_name}' deleted successfully"}
+
+
+@router.get("/agent/config", response_model=AgentConfig)
+async def get_agent_config(current_user: User = Depends(get_current_active_user)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)